### ISSUE TITLE
Sorting search results (7/16/202 email)

### ISSUE THREAD

**Issue by samhunter** [ucdavis-bioinformatics/NeuroMabSeq#33]
A search result for K89/34 will result in 6 results that seem sorted randomly. Jim suggests:

"I ran across this yesterday.  When I searched for “K89/34” I got this result.  The order returned is seemingly random.  It would be optimal to have them listed by convention. I propose first by category, and then within that category by name.  This would make the order K89/34.1 (category 1), followed by .2-.5 (category 3), and then K89/34R (category 7)."

In other words, can we sort search results by category ascending, name.

**Comment by keithgmitchell** [comment]
This should be an option on the filter (ordering) now

**Comment by keithgmitchell** [comment]
Actually seems this is not implemented with the API since it seems only one ordering field is allowed. Will work on this more for all filter views and see if I can come up with a solution
